### ISSUE TITLE
Expired certificate

### ISSUE THREAD

**Issue by MikaBerglund** [mikoskinen/blog#2]
Your Let's Encrypt certificate on your [blog site seems](https://mikaelkoskinen.net/) to have expired a couple of days ago.

**Comment by mikoskinen** [comment]
Thanks @MikaBerglund 👍 

Based on the logs, the Let's encrypt site extensions seems to have some issues binding the certificates with the site, as reported here: https://github.com/sjkp/letsencrypt-siteextension/issues/370.

**Comment by MikaBerglund** [comment]
I've noticed the same with the extension. Sometimes it fails to renew the certificate. That's why I always add my active e-mail address with the certificate request. This way, Let's Encrypt will send an alert before the certificate expires.

**Comment by mikoskinen** [comment]
Yep, that's a good feature to use. Not sure why I didn't have it enabled in this case. And  it was a good thing that the Pingdom check was pointing to the http-version of the site...  :) Though it is interesting that  even though there's a http->https redirect, Pingdom didn't seem to have any any issues with the site:

![image](https://user-images.githubusercontent.com/143570/83346642-0455b800-a327-11ea-97bc-dc86e9cf3266.png)

Lately I've been mainly using this one https://github.com/ohadschn/letsencrypt-webapp-renewer instead of https://github.com/sjkp/letsencrypt-siteextension for handling Let's Encrypt & App Service. It makes the life somewhat easier as I can use one app and its configuration to handle multiple App Services.

**Comment by MikaBerglund** [comment]
Perhaps Pingdom assumes that anything below 400 is an OK enough status code?
